lib/csvlint/cli.rb

```diff
@@ -83,10 +95,10 @@ def fetch_schema_tables(schema, options)
             return_error "#{source} not found"
           end
         end
-        valid &= validate_csv(source, schema, options[:dump_errors], nil, options[:werror], options[:verbose])
+        valid &= validate_csv(source, schema, options[:dump_errors], options[:format], options[:werror], options[:verbose])
```

Comment:
I think this change is enough to output the errors in a json format. Passing options[:json] here in this line outputs the errors in json format for me. But ya if we need output in xml, go ahead with these changes.

Reply:
Oops. Clicked the wrong button...

Yes, agreed.